### ISSUE TITLE
Buildscript does not work on arch linux

### ISSUE THREAD

**Issue by solonovamax** [pyroscope/rtorrent-ps#133]
It seems the build script is set up so that it _should_ work with arch linux, but doesn't.

The TL;DR of what happens is as follows:

on line 118 it executes `echo -n "$(lsb_release -sic 2>/dev/null || echo NonLSB)" | tr ' \n' '-'` which returns `Arch-n/a`.
Then, it goes into the switch case and matches the `*-n/a)` option. After it matches this option, it exits the switch case.
That means it never matches the `Arch-*)` option, which is what is needed to set the correct build dependencies from `( libncurses5-dev libncursesw5-dev libssl-dev zlib1g-dev libcppunit-dev locales )` to `( ncurses openssl cppunit )`, as arch has none of the dependencies for debian.

A simple solution would be to just move the `Arch-*)` case above the `*-n/a)` case, but that might cause unintended consequences, so I'll leave that up to you since you guys know what you're doing.
To build it on my system, I moved the arch case to the top and it worked fine, but you'd have to check other systems for compatibility.

Although, my _true_ preferred solution would be for you guys to just package a pkgbuild along with the buildscript, or allow the buildscript to generate a pkgbuild, so I could do something like `./build.sh configureArch` `make` `make install`, and then it's done. Of course, I can understand that you might not want to create such a script as it'll then require maintaining which is more effort.

**Comment by pyroscope** [comment]
The links to the existing pkg-builds are in the docs.

**Comment by solonovamax** [comment]
But the script should also work for arch?
I don't see any reason why it shouldn't...
